fs/attr.c

```diff
@@ -174,10 +180,17 @@ int setattr_prepare(struct user_namespace *mnt_userns, struct dentry *dentry,
 		if (!inode_owner_or_capable(mnt_userns, inode))
 			return -EPERM;
 #endif
```

Comment:
@anodos325 could you please take a look at some updates into this file (fs/attr.c). I resolved a few conflicts here during the merge.

Here is the [link](https://lore.kernel.org/all/20220613111517.2186646-1-brauner@kernel.org/) to the changeset that caused the conflict.